incubator/binding-risingwave.spec/src/main/scripts/io/aklivity/zilla/specs/binding/risingwave/schema/risingwave.schema.patch.json

```diff
@@ -83,7 +83,9 @@
                                         }
                                     },
                                     "additionalProperties": false
-                                }
+                                },
```

Comment:
The `language` above should be `"const": "java"` too, right?
Since we only support the value "java" if present for now.